LEGO1/lego/legoomni/include/legocarbuild.h

```diff
@@ -55,15 +60,13 @@ typedef LegoVehicleBuildState LegoDuneCarBuildState;
 typedef LegoVehicleBuildState LegoJetskiBuildState;
 
 // VTABLE: LEGO1 0x100d6658
+// VTABLE: BETA10 0x101bb880
 // SIZE 0x34c
 class LegoCarBuild : public LegoWorld {
 public:
 	LegoCarBuild();
 	~LegoCarBuild() override;
 
-	MxLong Notify(MxParam& p_param) override; // vtable+0x04
```

Comment:
This should be reverted, otherwise the virtual functions are no longer declared in order